BD-1937: Update AMP clients (#4905)
braze-inc/braze-docs: _docs/_user_guide/message_building_by_channel/email/amphtml.md

| AMP Email Elements | A compelling AMP Email includes the strategic use of various components.<br>Check out the Essentials tab in the [Components](#components) section below. |
{: .reset-td-br-1 .reset-td-br-2}

### Supported clients

Before you can send AMP Emails to users, you must register with our clients. The registration process involves sending a test AMPHTML email to get approved. Approval times vary client to client. Follow the register links for more information.

| Client | Register Link |
| ------ | -------- |
| Gmail | [Google](https://developers.google.com/gmail/ampemail/register) |

> | FairEmail | [FairEmail](https://email.faircode.eu/)

Author: ```suggestion
| FairEmail | [FairEmail](https://email.faircode.eu/) |
```

> | Yahoo | [Yahoo](https://senders.yahooinc.com/amp/)

Author: ```suggestion
| Yahoo | [Yahoo](https://senders.yahooinc.com/amp/) |
```

| Mail.ru | [Mail.ru](https://postmaster.mail.ru/amp/) |

> For a full list of supported platforms, click [here](https://amp.dev/support/faq/email-support). 

Author: ```suggestion
For a full list of supported platforms, visit the [AMP documentation](https://amp.dev/support/faq/email-support). 
```
